packages/twenty-front/src/modules/action-menu/actions/record-actions/single-record/hooks/useExportNoteAction.ts

```diff
@@ -0,0 +1,76 @@
+import { SingleRecordActionHookWithObjectMetadataItem } from '@/action-menu/actions/types/SingleRecordActionHook';
+import { recordStoreFamilyState } from '@/object-record/record-store/states/recordStoreFamilyState';
+import { BlockNoteEditor } from '@blocknote/core';
+import {
+  DOCXExporter,
+  docxDefaultSchemaMappings,
+} from '@blocknote/xl-docx-exporter';
+import {
+  PDFExporter,
+  pdfDefaultSchemaMappings,
+} from '@blocknote/xl-pdf-exporter';
+import * as ReactPDF from '@react-pdf/renderer';
+import { Buffer } from 'buffer';
+import { Packer } from 'docx';
+import { saveAs } from 'file-saver';
+import { useRecoilValue } from 'recoil';
+import { isDefined } from 'twenty-ui';
+
```

Comment:
Could you please fix the CI? Make sure linter/tests are always passing. Thanks!

<img width="1113" alt="Screenshot 2024-12-29 at 15 41 13" src="https://github.com/user-attachments/assets/c62c34f9-dc3a-43a6-ba9b-78337933188c" />

Reply:
Thank you for pointing that out! The CI failure appears to be due to exportNoteToDocX being defined but not yet used. I haven’t implemented it at this stage because, given the current code structure, doing so would require creating two nearly identical hooks that differ only in the exportTo argument (pdf vs. docx). Since issue #9272 looks like it would address this elegantly, I’ll comment out exportNoteToDocX for now to ensure the linter and tests pass. Please let me know if there’s a preferred approach you’d like me to take here!

Reply:
Resolved in recent commit.